web/modules/custom/asset_release/tests/src/Functional/AssetReleaseWorkflowTest.php

```diff
@@ -191,8 +190,7 @@ public function testWorkflow() {
         $content->save();
 
         // Override the user to be checked for the allowed transitions.
-        $this->setCurrentUser($this->{$user_var});
-        $actual_transitions = $content->field_isr_state->first()->getTransitions();
+        $actual_transitions = $this->workflowHelper->getAvailableTransitions($content, $this->{$user_var});
```

Comment:
Minor remark: it is not needed to wrap this in curly braces, `$this->$user_var` is legal PHP.